widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -1537,6 +1537,19 @@ button#schedule-dialog-select-button {
 }
 /* endregion GWT DialogBox, PromptDialogBox RESPONSIVE */
 
+/* region Responsive Properties dialog - Start */
```

Comment:
Isn't the name of the dialog "File Properties"? If so, I think you should update both the comments and the CSS class itself to reflect that.

Reply:
The Dialog class is called "FilePropertiesDialog" but it is the same Dialog for the Folders as well. I felt keeping the name file in the client side was not generic enough.

Reply:
I see. It would be a `FileOrFolderPropertiesDialog`. Still, lacking perfection, `file-properties-dialog` is both closer to that, than the generic `properties-dialog` is, and it simply follows the imprecision already present in the Java class name.

Reply:
addressed

Reply:
Please, change the name to the new one in the region comments as well (I had mentioned that in the original comment).

Reply:
addressed